open_spiel/games/mancala_test.cc

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 DeepMind Technologies Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "open_spiel/spiel.h"
+#include "open_spiel/tests/basic_tests.h"
+
+namespace open_spiel {
+namespace mancala {
+namespace {
+
+namespace testing = open_spiel::testing;
+
+void BasicMancalaTests() {
+  
+}
```

Comment:
Please add some basic tests. At the bare minimum, you should do the simulation tests (see e.g. breakthrough_test.cc). 

It's always better to have also have a few Mancala-specific tests too (see e.g. backgammon_test.cc), especially f there are any tricky rules situations that might occur.